docs/tutorials/operations.md

```diff
@@ -108,6 +108,73 @@ domain
 [known bug](https://github.com/zalando/skipper/issues/1024), if you
 combine it with custom routes.
 
+Alternatively, you can use Kubernetes East West Range feature. Use the
```

Comment:
Maybe we should add a subsection for all the "more features"

Reply:
great idea, done!